docs/supported_formats/edax.rst

```diff
@@ -10,43 +10,32 @@ spectrum image is loaded from the corresponding ``.ipr`` and ``.spc`` files
 stored in the same directory, or from specific files indicated by the user.
 If these calibration files are not available, the data from the ``.spd``
 file will still be loaded, but with no spatial or energy calibration.
+
 When using `HyperSpy <https://hyperspy.org>`_, if elemental information has been
-defined in the spectrum image, those
-elements will automatically be added to the signal loaded by HyperSpy.
+defined in the spectrum image, those elements will automatically be added to the
+signal loaded by HyperSpy.
 
-In HyperSpy, if an ``.spd`` file is loaded, the result will be a Hyperspy
+In HyperSpy, if an ``.spd`` file is loaded, the result will be a HyperSpy
 ``EDSSEMSpectrum`` map, and the calibration will be loaded from the appropriate
```

Comment:
Note for later, before it gets lost: for a separate PR, it would be good to review references to HyperSpy API objects and link it correctly - it may need further improvement to the HyperSpy API, not sure if the attribute object are well documented at the moment in HyperSpy documentation.

Reply:
Yes, we should create an issue for that. But the new documentation should maybe be released for that?

Also I found some missing labels when referencing from LumiSpy to HyperSpy, which currently require full links instead of proper references.

Reply:
If this is the case, we should change the branch of the development version on readthedocs.org to use the `RELEASE_next_major` branch instead of the `RELEASE_next_minor` branch, as it make more sense to work from that at the moment. By doing this, we don't need to rely on a release version and we can tweak labels iteratively if necessary, etc.
I don't have the permission the readthedocs.org to change it, @francisco-dlp, can you please change this setting on readthedocs.org?